### PR TITLE
Remove email normalization

### PR DESCRIPTION
**Pull request by estellecomment**
Email normalization does a lot of weird things : see "normalizeEmail" in https://github.com/validatorjs/validator.js#sanitizers

In particular it removes dots in gmail adresses : aaa.bbb@gmail.com is normalized to aaabbb@gmail.com (which is an unknown adress for us). 
This is breaking login for gmail adresses with dots in them.

In another PR, we could look at lowercasing emails... But first we have to check the emails in db are all lowercase, so it's not trivial (other PR)

### CODE REVIEW

File: controllers/loginController.js [hunk 1]
```diff
@@ -92,15 +91,21 @@ module.exports.postLogin = async function postLogin(req, res) {
     return res.redirect('/psychologue/login');
   }
   const email = req.body.email;
+  console.log('email', email)
```
Comment (polomarcus) [inline controllers/loginController.js:94]:
```suggestion
```

File: controllers/loginController.js [hunk 2]
```diff
@@ -12,7 +12,6 @@ const config = require('../utils/config');
 module.exports.emailValidators = [
   check('email')
     .isEmail()
-    .normalizeEmail() //sanitize input
```
Comment (polomarcus) [inline controllers/loginController.js:15]: my bad !

File: controllers/loginController.js [hunk 3]
```diff
@@ -92,15 +91,21 @@ module.exports.postLogin = async function postLogin(req, res) {
...
 
     if( emailExist ) {
       const token = generateToken();
+//      console.log('token', token)
       const loginUrl = generateLoginUrl();
+ //     console.log('loginUrl', loginUrl)
       await sendLoginEmail(email, loginUrl, token);
+   //   console.log('sent mail')
       await saveToken(email, token);
+     // console.log('token saved')
```
Comment (polomarcus) [inline controllers/loginController.js:108]:
```suggestion
```

File: controllers/loginController.js [hunk 4]
```diff
@@ -92,15 +91,21 @@ module.exports.postLogin = async function postLogin(req, res) {
...
     const emailExist = await dbPsychologists.getPsychologistByEmail(email);
+//    console.log('emailExist', emailExist)
 
     if( emailExist ) {
       const token = generateToken();
+//      console.log('token', token)
       const loginUrl = generateLoginUrl();
+ //     console.log('loginUrl', loginUrl)
       await sendLoginEmail(email, loginUrl, token);
+   //   console.log('sent mail')
```
Comment (polomarcus) [inline controllers/loginController.js:106]:
```suggestion
```

File: controllers/loginController.js [hunk 5]
```diff
@@ -92,15 +91,21 @@ module.exports.postLogin = async function postLogin(req, res) {
...
 
   try {
     const emailExist = await dbPsychologists.getPsychologistByEmail(email);
+//    console.log('emailExist', emailExist)
 
     if( emailExist ) {
       const token = generateToken();
+//      console.log('token', token)
       const loginUrl = generateLoginUrl();
+ //     console.log('loginUrl', loginUrl)
```
Comment (polomarcus) [inline controllers/loginController.js:104]:
```suggestion
```

File: controllers/loginController.js [hunk 6]
```diff
@@ -92,15 +91,21 @@ module.exports.postLogin = async function postLogin(req, res) {
...
   const email = req.body.email;
+  console.log('email', email)
 
   try {
     const emailExist = await dbPsychologists.getPsychologistByEmail(email);
+//    console.log('emailExist', emailExist)
 
     if( emailExist ) {
       const token = generateToken();
+//      console.log('token', token)
```
Comment (polomarcus) [inline controllers/loginController.js:102]:
```suggestion
```

File: controllers/loginController.js [hunk 7]
```diff
@@ -92,15 +91,21 @@ module.exports.postLogin = async function postLogin(req, res) {
     return res.redirect('/psychologue/login');
   }
   const email = req.body.email;
+  console.log('email', email)
 
   try {
     const emailExist = await dbPsychologists.getPsychologistByEmail(email);
+//    console.log('emailExist', emailExist)
```
Comment (polomarcus) [inline controllers/loginController.js:98]:
```suggestion
```

**Review by polomarcus — APPROVED** [review]

**Comment by jdauphant** [comment]
il faut normaliser la comparaison si vous voulez utiliser "normalizeEmail" (garder en base la saisie exacte de utilisateur et normaliser au moment de comparer à la connexion)

**Pull request merged**